applications_tests/gls_sharp_navier_stokes_2d/couette2d_gls_poisson.output

```diff
@@ -7,26 +7,26 @@ Running on 1 MPI rank(s)...
 Steady iteration :        1/3
 *****************************************************************
 div u : 0
-L2 error velocity : 0.00418651 L2 error pressure: 0
+L2 error velocity : 0.171083 L2 error pressure: 0
 
 *****************************************************************
 Steady iteration :        2/3
 *****************************************************************
    Number of active cells:       4096
    Number of degrees of freedom: 12675
    Volume of triangulation:      4
-div u : 5.12312e-10
-L2 error velocity : 0.00109947 L2 error pressure: 1.05594e-08
```

Comment:
What happened here again? Why is the error freaking huge all of a sudden for the velocity field?